### PR TITLE
multi-cloud deploy need

### PR DESCRIPTION
**Pull request by lijuncloud**
multi-cloud deploy need

### CODE REVIEW

File: api/pkg/utils/config/config.go [hunk 1]
```diff
@@ -27,7 +27,48 @@ import (
 const (
 	ConfKeyName = iota
 	ConfDefaultValue
+	AuthPath="/etc/multi/multi.conf"
 )
+func GetAuthStrateger() string{
```
Comment (leonwanghui) [inline api/pkg/utils/config/config.go]: s/GetAuthStateger/GetAuthStrategy

Comment (lijuncloud) [reply api/pkg/utils/config/config.go]: have replace it

File: api/pkg/filters/auth/auth.go [hunk 2]
```diff
@@ -35,28 +34,27 @@ func NewNoAuth() AuthBase {
...
 	ctx.IsAdmin = ctx.TenantId == c.NoAuthAdminTenantId
 	chain.ProcessFilter(req, resp)
 }
 
 func FilterFactory() restful.FilterFunction {
 	var auth AuthBase
-	log.Log("authStrateger:\n")
-	log.Log(os.Getenv("AUTH_AuthStrategy"))
-	switch os.Getenv("AUTH_AuthStrategy") {
+	s := cf.GetAuthStrateger()
```
Comment (leonwanghui) [inline api/pkg/filters/auth/auth.go]: I haven't seen the update, it should be `GetAuthStategy`

Comment (lijuncloud) [reply api/pkg/filters/auth/auth.go]: > I haven't seen the update, it should be `GetAuthStategy`
   I Have update it, thanks for your suggestion

File: api/pkg/utils/config/config.go [hunk 3]
```diff
@@ -27,7 +27,48 @@ import (
 const (
 	ConfKeyName = iota
 	ConfDefaultValue
+	AuthPath="/etc/multi/multi.conf"
 )
+func GetAuthStateger() string{
```
Comment (leonwanghui) [inline api/pkg/utils/config/config.go]: Have you updated it?

**Comment by xxwjj** [comment]
good job

**Pull request merged**